### ISSUE TITLE
release: v19.2.10

### ISSUE THREAD

**Issue by apantel** [cockroachdb/cockroach#52630]
Candidate SHA: WIP EDIT ME WIP.
Deployment status: WIP EDIT ME WIP.
Qualification Suite: WIP EDIT ME WIP.
Nightly Suite: WIP EDIT ME WIP.
 
Admin UI for Qualification Clusters:
- Old Version: WIP EDIT ME WIP.
- New Version: WIP EDIT ME WIP.
 
[Release process checklist](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process)
 
Prep date: `WIP fill in date to roll out: usually Monday, a week before the release WIP`
- [ ] [Pick a SHA](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-PickingaSHA)
  - [ ] fill in `Candidate SHA` above
  - [ ] email thread on releases@
- [ ] [Tag the provisional SHA](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-TagtheprovisionalSHA)
- [ ] [Publish provisional binaries](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Publishtheprovisionalbinaries)
- [ ] Ack security@ and release-engineering-team@ on the generated AWS S3 bucket write Alert to confirm these writes were part of a planned release (Just reply on the email received alert email acking that this was part of the release process)
 
Release Qualification
- [ ] [Check binaries](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Checkbinaries)
- [ ] [Deploy to test clusters](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-DeploytoTestClusters)
- [ ] [Verify node crash reports](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Verifynodecrashreportsappearinsentry.io)
- [ ] [Start release qualification suite](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-RuntheSuiteofNightlyRoachtests)
- [ ] [Start nightly suite](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-RuntheSuiteofNightlyRoachtests)
- [ ] fill in all WIP elements above (Nightly Suite, Admin UI, etc.)
 
One day after prep date:
- [ ] [Get signoff on roachtest failures](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/328859690/Release+Qualification#Roachtest-Failure-Signoff)
 
- [ ] [Keep an eye on clusters](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/328859690/Release+Qualification#Monitoring-the-Qualification-Clusters) until release date. Do not proceed below until the release date.
 
Release date: `WIP fill in date: usually Monday WIP`
- [ ] [Check cluster status](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/328859690/Release+Qualification#Monitoring-the-Qualification-Clusters)
- [ ] [Tag release](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-11.Tagtherelease)
- [ ] [Bless provisional binaries](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Blesstheprovisionalbinaries)
- [ ] If applicable, [update the map that roachtests use to map a version to a previous version, to reference the newly tagged version](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/test_runner.go#L1145)
 
- For production or stable releases in the latest major release series
  - [ ] Update the [Homebrew Formula](https://github.com/cockroachdb/homebrew-tap/blob/master/Formula/cockroach.rb)
  - [ ] Update the image tag in our [Orchestrator configurations](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Orchestratorconfigurations)
 
- For production or stable releases
  - [ ] Announce version to the [registration cluster](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-AnnounceVersionToRegCluster)
 
- [ ] Update docs
- [ ] External communications for release
 
Cleanup:
- [ ] Clean up provisional tag from repository
- [ ] Destroy roachprod clusters

**Comment by RaduBerinde** [comment]
New issue #52841, closing this one.
